code/game/objects/structures/dresser.dm

```diff
@@ -5,6 +5,8 @@
 	icon_state = "dresser"
 	density = 1
 	anchored = 1
+	burn_state = FLAMMABLE
+	burntime = 25
```

Comment:
It's more of a matter of principle.

Not being able to telcomms equipment generally isn't going to matter most rounds, but as matter of principle, something that's generally of decent importance needs to be createable or immune to easy ways of destroying it.